tests/integration/test_connection.py

```diff
@@ -1,5 +1,5 @@
 from pyserum.connection import get_live_markets, get_token_mints
```

Comment:
I am wondering if it will make sense to put the dex related integration test into the `tests/market` directory? Since I am expecting we will have other integration tests, say for the swap.

And also maybe we should rename `tests/market` -> `tests/dex` to make it conform to the source.

Reply:
I think we can keep the integration tests all in the same directory. Let's just name the files properly for now.